src/vsr/journal.zig

```diff
@@ -1525,7 +1525,8 @@ pub fn JournalType(comptime Replica: type, comptime Storage: type) type {
                     assert(prepare.?.operation == .reserved);
                     assert(header.?.checksum == prepare.?.checksum);
                     assert(
-                        header.?.checksum == Header.Prepare.reserved_operation(cluster, slot.index).checksum,
+                        header.?.checksum ==
+                            Header.Prepare.reserved_operation(cluster, slot.index).checksum,
```

Comment:
Oh, maybe `Header.Prepare.reserve(cluster, slot.index)`? As a verb?

Reply:
`reserve` as a verb sounds more like an acquire/release mechanism.

maybe `as_reserved()`, `from_reserved()`, or the other way around, renaming the field `reserved` to something else like `reserved_bytes` or `padding`.